pl_engine/pl_server/src/main/java/com/cubrid/jsp/ExecuteThread.java

```diff
@@ -480,4 +485,20 @@ private void sendError(String exception) throws IOException {
         resultBuffer = packer.getBuffer();
         writeBuffer(resultBuffer);
     }
+
+    private void sendAuthCommand(int command, String authName) throws Exception {
+        AuthInfo info = new AuthInfo(command, authName);
+        CUBRIDPacker packer = new CUBRIDPacker(ByteBuffer.allocate(128));
+        packer.packInt(RequestCode.REQUEST_CHANGE_AUTH_RIGHTS);
+        info.pack(packer);
+        Context.getCurrentExecuteThread().sendCommand(packer.getBuffer());
+
+        ByteBuffer responseBuffer = Context.getCurrentExecuteThread().receiveBuffer();
+        CUBRIDUnpacker unpacker = new CUBRIDUnpacker(responseBuffer);
+        /* read header, dummy */
+        Header header = new Header(unpacker);
```

Comment:
Local variables header and responseCode are not used after the assignments.

Reply:
They are dummy data and not used now.